.github/dependabot.yaml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/images"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  versioning-strategy: lockfile-only
+- package_manager: "docker"
+  directory: "/"
+  update_schedule: "weekly"
```

Comment:
```suggestion
```

Reply:
Let's experiment with this later, but it was not documented what this meant, so I wanted to leave it out for now. I assume it means to inspect Dockerfile and bump FROM images etc though, but could not find dependabot.yml documentation about this.